vscode-extension/package.json

```diff
@@ -2,7 +2,13 @@
   "name": "relay",
   "displayName": "Relay GraphQL",
   "version": "0.0.1",
-  "description": "Relay LSP Client for VSCode",
+  "description": "Relay-powered IDE experience",
```

Comment:
Most big extensions end up on vscodium and others via https://open-vsx.org/ - so it doesn't hurt to remove the vs code mention now